webapp/sources/ldap-inventory/inventory-repository/src/main/scala/com/normation/inventory/ldap/core/InventoryMapper.scala

```diff
@@ -775,6 +773,16 @@ class InventoryMapper(
         win.setOpt(productId, A_WIN_ID, { x: String => x })
         win
     }
+    root
+  }
+
+  def treeFromNode(server:NodeInventory) : LDAPTree = {
```

Comment:
it needs at least a comment, perhaps even a parameter `includeProcess: Boolean = false`. Else it's sure that one day we will wonder why our nodes are missing process in LDAP